utils/create-icons-map.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const svgFolder = '../assets/icons/';
+const iconMapFile = '../src/iconsMap.json';
+const svgCleaner = new RegExp('("fill:.*;")|(fill=".*")|(id=\'.*\' )|(width=".*")|height=(".*")', 'g');
```

Comment:
Why isn't height inside of the ( )? It won't be caught in the regex match. 

And this one: (id=\'.*\' )? As far as I know, everything is in double quotes, but we need IDs anyway right now, because we are using the <use/> tag that references by ID.